common/services/prismic/parsers.js

```diff
@@ -544,6 +547,26 @@ export function parseOnThisPage(fragment: PrismicFragment[]): Link[] {
     });
 }
 
+export function parseMediaObjectList(
+  fragment: PrismicFragment[]
+): Array<MediaObjectType> {
+  return fragment.map(mediaObject => {
+    if (mediaObject) {
+      // make sure we have the content we require
+      const title = mediaObject.title.length ? mediaObject?.title : null;
```

Comment:
Is this searching for empty strings? If so we could stick it in a method named that.

We're also trying to use `undefined` over `null` (I'll dig out some literature on this, but basically it's to stop having two empty type values).